// __mf__virtual/signing__loadShare__vue_mf_2_router__loadShare__.mjs

    import "__mf__virtual/signing__prebuild__vue_mf_2_router__prebuild__.js";
    
    
const __mfPromiseGlobalKey = "__mf_init____mf__virtual/signing__mf_v__runtimeInit__mf_v__.js__";
let __mfPromiseState = globalThis[__mfPromiseGlobalKey];
if (!__mfPromiseState) {
  let initResolve, initReject;
  const initPromise = new Promise((re, rj) => {
    initResolve = re;
    initReject = rj;
  });
  __mfPromiseState = globalThis[__mfPromiseGlobalKey] = {
    initPromise,
    initResolve,
    initReject,
  };
  if (typeof window === 'undefined') {
    initResolve({
      loadRemote: function() { return Promise.resolve(undefined); },
      loadShare: function() { return Promise.resolve(undefined); },
    });
  }
}
const initPromise = __mfPromiseState.initPromise;

    
    const res = initPromise.then(runtime => runtime.loadShare("vue-router", {
      customShareInfo: {shareConfig:{
        singleton: true,
        strictVersion: false,
        requiredVersion: "^4.5.0"
      }}
    }))
    const exportModule = await res.then((factory) => (typeof factory === "function" ? factory() : factory))
    export default exportModule;
    const { NavigationFailureType: __mf_0, RouterLink: __mf_1, RouterView: __mf_2, START_LOCATION: __mf_3, createMemoryHistory: __mf_4, createRouter: __mf_5, createRouterMatcher: __mf_6, createWebHashHistory: __mf_7, createWebHistory: __mf_8, isNavigationFailure: __mf_9, loadRouteLocation: __mf_10, matchedRouteKey: __mf_11, onBeforeRouteLeave: __mf_12, onBeforeRouteUpdate: __mf_13, parseQuery: __mf_14, routeLocationKey: __mf_15, routerKey: __mf_16, routerViewLocationKey: __mf_17, stringifyQuery: __mf_18, useLink: __mf_19, useRoute: __mf_20, useRouter: __mf_21, viewDepthKey: __mf_22 } = exportModule;
    export { __mf_0 as NavigationFailureType, __mf_1 as RouterLink, __mf_2 as RouterView, __mf_3 as START_LOCATION, __mf_4 as createMemoryHistory, __mf_5 as createRouter, __mf_6 as createRouterMatcher, __mf_7 as createWebHashHistory, __mf_8 as createWebHistory, __mf_9 as isNavigationFailure, __mf_10 as loadRouteLocation, __mf_11 as matchedRouteKey, __mf_12 as onBeforeRouteLeave, __mf_13 as onBeforeRouteUpdate, __mf_14 as parseQuery, __mf_15 as routeLocationKey, __mf_16 as routerKey, __mf_17 as routerViewLocationKey, __mf_18 as stringifyQuery, __mf_19 as useLink, __mf_20 as useRoute, __mf_21 as useRouter, __mf_22 as viewDepthKey };
