// __mf__virtual/signing__loadShare__pinia__loadShare__.mjs

    import "__mf__virtual/signing__prebuild__pinia__prebuild__.js";
    
    
const __mfPromiseGlobalKey = "__mf_init____mf__virtual/signing__mf_v__runtimeInit__mf_v__.js__";
let __mfPromiseState = globalThis[__mfPromiseGlobalKey];
if (!__mfPromiseState) {
  let initResolve, initReject;
  const initPromise = new Promise((re, rj) => {
    initResolve = re;
    initReject = rj;
  });
  __mfPromiseState = globalThis[__mfPromiseGlobalKey] = {
    initPromise,
    initResolve,
    initReject,
  };
  if (typeof window === 'undefined') {
    initResolve({
      loadRemote: function() { return Promise.resolve(undefined); },
      loadShare: function() { return Promise.resolve(undefined); },
    });
  }
}
const initPromise = __mfPromiseState.initPromise;

    
    const res = initPromise.then(runtime => runtime.loadShare("pinia", {
      customShareInfo: {shareConfig:{
        singleton: true,
        strictVersion: false,
        requiredVersion: "^2.2.2"
      }}
    }))
    const exportModule = await res.then((factory) => (typeof factory === "function" ? factory() : factory))
    export default exportModule;
    const { MutationType: __mf_0, PiniaVuePlugin: __mf_1, acceptHMRUpdate: __mf_2, createPinia: __mf_3, defineStore: __mf_4, disposePinia: __mf_5, getActivePinia: __mf_6, mapActions: __mf_7, mapGetters: __mf_8, mapState: __mf_9, mapStores: __mf_10, mapWritableState: __mf_11, setActivePinia: __mf_12, setMapStoreSuffix: __mf_13, skipHydrate: __mf_14, storeToRefs: __mf_15 } = exportModule;
    export { __mf_0 as MutationType, __mf_1 as PiniaVuePlugin, __mf_2 as acceptHMRUpdate, __mf_3 as createPinia, __mf_4 as defineStore, __mf_5 as disposePinia, __mf_6 as getActivePinia, __mf_7 as mapActions, __mf_8 as mapGetters, __mf_9 as mapState, __mf_10 as mapStores, __mf_11 as mapWritableState, __mf_12 as setActivePinia, __mf_13 as setMapStoreSuffix, __mf_14 as skipHydrate, __mf_15 as storeToRefs };
